spring-amqp/src/main/java/org/springframework/amqp/core/Address.java

```diff
@@ -39,14 +39,25 @@
  */
 public class Address {
 
+	/**
+	 * Use this value in {@code RabbitTemplate#setReplyAddress(String)} to use a temporary reply queue
+	 * for each request, regardless of whether the broker supports direct reply-to.
+	 */
+	public static final String TEMPORARY_REPLY_QUEUE_TOKEN = "##TEMPORARY_QUEUE##";
```

Comment:
Ah, now it makes more sense; thanks.

The reason I was going to backport it is because I assumed you would need it before May (the planned release for 1.6).

If you can work around it with a named queue with 1.5 we can just apply it to 1.6.

But I have no issues putting it in 1.5.4 (sans namespace change) if you want.